python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -63,53 +63,50 @@ class RandomForestClassifier(DelayedPredictionMixin):
 
     Parameters
     -----------
-    n_estimators : int (default = 10)
+    n_estimators: int (default = 10)
                    total number of trees in the forest (not per-worker)
-    handle : cuml.Handle
-             If it is None, a new one is created just for this class.
+    handle: cuml.Handle
```

Comment:
I'm surprised about the change to the colons here... the old code seems to follow the numpydoc style correct: https://numpydoc.readthedocs.io/en/latest/format.html (with spaces). Do you know what's going on there?

Reply:
without the space,ie with the current updated code, the docs look like :
"
n_estimators: int (default = 100)
Number of trees in the forest. (Default changed to 100 in cuML 0.11)

handle: cuml.Handle
If it is None, a new one is created just for this class.
"

If i leave the space in, ie in the numpydoc style. the doc looks like:
"
**n_estimators**int (default = 100)
Number of trees in the forest. (Default changed to 100 in cuML 0.11)

**handle**cuml.Handle
If it is None, a new one is created just for this class.
"

Reply:
Hmm... why does it not appear messed up in the online docs with this formatting? https://docs.rapids.ai/api/cuml/nightly/

Reply:
I used `cuml/docs/build/html/api.html`  file to check the format of the documentation.